Error '_io.TextIOWrapper' object has no attribute 'reconfigure' for python 3.6

A recent change has introduced a failure for python < 3.7:

https://github.com/EnricoMi/publish-unit-test-result-action/blob/master/python/publish/github_action.py#L16

The `reconfigure` method was added in python 3.7, so causes issues when using python 3.6

```
Publish Unit Test Results
  Traceback (most recent call last):
    File "/home/runner/work/_actions/EnricoMi/publish-unit-test-result-action/v1/composite/../python/publish_unit_test_results.py", line 329, in <module>
      gha = GithubAction()
    File "/home/runner/work/_actions/EnricoMi/publish-unit-test-result-action/v1/python/publish/github_action.py", line 16, in __init__
      file.reconfigure(encoding='utf-8')
  AttributeError: '_io.TextIOWrapper' object has no attribute 'reconfigure'
  Error: Process completed with exit code 1.
```

--- a/python/publish/github_action.py
+++ b/python/publish/github_action.py
@@ -11,7 +11,8 @@ class GithubAction:
     def __init__(self, file: Optional[TextIOWrapper] = None):
         if file is None:
             file = sys.stdout
-            if isinstance(file, TextIOWrapper):
+            # pre Python 3.7, TextIOWrapper does not have reconfigure
+            if isinstance(file, TextIOWrapper) and hasattr(file, 'reconfigure'):
                 # ensure we have utf8 encoding, the default encoding of sys.stdout on Windows is cp1252
                 file.reconfigure(encoding='utf-8')
 

--- a/python/publish/github_action.py
+++ b/python/publish/github_action.py
@@ -1,3 +1,4 @@
+import logging
 import os
 import sys
 from io import TextIOWrapper
@@ -74,5 +75,9 @@ class GithubAction:
         params = ','.join([f'{key}={str(value)}'
                            for key, value in params.items()])
         params = f' {params}' if params else ''
-        file.write(f'::{command}{params}::{value}')
-        file.write(os.linesep)
+
+        try:
+            file.write(f'::{command}{params}::{value}')
+            file.write(os.linesep)
+        except Exception as e:
+            logging.error(f'Failed to forward command {command} to GithubActions: {e}')

